sqitch/deploy/update_view_employee_project_responsible.sql

```diff
@@ -0,0 +1,29 @@
+-- Deploy floq:update_view_employee_project_responsible to pg
+
+BEGIN;
+
+CREATE OR REPLACE VIEW employee_project_responsible AS
+SELECT most_staffed.employee as employee,
+       projects.responsible as project_responsible
+FROM (SELECT employee,
+             project
+      FROM (SELECT employee,
+                   project,
+                   total,
+                   row_number() OVER (PARTITION BY employee ORDER BY total DESC) as rank
+            FROM (SELECT employee,
+                         project,
+                         count(employee) AS total
+                  FROM staffing
+                  WHERE date > (now() - '22 days'::interval)
+                    AND date < (now() + '21 days'::interval)
+                    AND project != 'FER1000'
```

Comment:
Er ikke sikker men burde vi joine inn `absense_reasons` og sjekke alle i stedet for å hardkode inn FER1000?

Reply:
Skjønner tanken, men tror ikke vi trenger det.
Man staffes gjerne ikke opp på koder med kortere horisont, ala egenmelding. Og hvis man er staffet på noe med lang horisont, typ foreldrepermisjon, så kan det være greit at man har en ansvarlig som får de genererte oppgavene knyttet til seg

Reply:
@magnedavidsen Noen kommentar?

Reply:
tror jeg er enig i at dette gjelder kun ferie, ja